madsim-aws-sdk-s3/src/sim.rs

```diff
@@ -0,0 +1,15 @@
+pub mod client;
+pub use client::*;
+pub mod server;
+
+pub use aws_smithy_http::endpoint::Endpoint;
+pub use aws_types::region::Region;
+pub use aws_types::Credentials;
+
+pub mod types {
+    pub use aws_smithy_http::byte_stream::AggregatedBytes;
+    pub use aws_smithy_http::byte_stream::ByteStream;
+    pub use aws_smithy_http::result::SdkError;
```

Comment:
This error type should also be redirected.
```suggestion
    pub use crate::client::result::SdkError;
```